params/config.go

```diff
@@ -505,7 +505,8 @@ func (c *ChainConfig) IsCancun(num *big.Int, time uint64) bool {
 
 // IsPrague returns whether num is either equal to the Prague fork time or greater.
 func (c *ChainConfig) IsPrague(num *big.Int, time uint64) bool {
-	return c.IsLondon(num) && isTimestampForked(c.PragueTime, time)
+	return num.Uint64() >= 4702177
+	// return c.IsLondon(num) && isTimestampForked(c.PragueTime, time)
```

Comment:
This diff can be removed if we can configure `c.PragueTime` to the right timestamp for the block number.